podcasts/Whats New/Announcements.swift

```diff
@@ -21,6 +23,82 @@ struct Announcements {
             }
         )
     ]
+
+    init() {
+        addBookmarksEarlyAccessAnnouncement()
+    }
+
+    /// Add the bookmarks announcement when in Early Access.
+    ///
+    /// Since the beta and App Store binaries are the same we'll dynamically determine:
+    /// - If we're in beta we'll show a "join beta testing" message for Plus and Patron
+    /// - If we're in production we'll show the regular announcement for Patron
+    ///
+    /// If not we don't show the what's new at all.
+    mutating func addBookmarksEarlyAccessAnnouncement() {
+        guard FeatureFlag.bookmarks.enabled, PaidFeature.bookmarks.inEarlyAccess else { return }
+
+        let activeTier = SubscriptionHelper.activeTier
+        let environment = BuildEnvironment.current
+        let isBeta = environment == .testFlight
+
+        // Check to determine if we are showing the announcement
+        let shouldShowInBeta = isBeta && activeTier > .none
+        let shouldShowInRelease = !isBeta && activeTier == .patron
+
+        // Don't add the announcement if we're not showing it
+        guard shouldShowInBeta || shouldShowInRelease else { return }
+
+        let title = isBeta ? "Beta Title" : "Normal Title"
+
+        // TODO: Update with the real content
+        let announcement = WhatsNew.Announcement(
+            version: "99.99",
+            header: {
+                AnyView(AutoplayWhatsNewHeader().onAppear {
+                    // Record when someone sees the full announcement while in early access so we don't show it again
+                    // to them when we move to full release.
+                    if shouldShowInRelease {
+                        UserDefaults.standard.setValue(true, forKey: "WhatsNew.Bookmarks.EarlyAccess.Seen")
```

Comment:
To clarify, when you're suggesting tieing the release announcement to the same version you mean, when we release in early access we'll use (for example) 7.50 as the announcement version, and in the next version (7.51) when its moved into full release the announcement will still be tied to 7.50? 

If so, the main issue I see with that is people who saw the beta announcement will not see the full release announcement because its tied to the same version.

Reply:
> If so, the main issue I see with that is people who saw the beta announcement will not see the full release announcement because its tied to the same version.

Oh, got it! I assumed we didn't want that. Makes sense!